packages/financial-health-calculator/src/data-fetchers/open-collective.ts

```diff
@@ -56,6 +56,7 @@ export const fetchOpenCollectiveData = (packageName: string): Promise<OpenCollec
 
   return fetch('https://api.opencollective.com/graphql/v2', options)
     .then((response) => response.json())
+    .catch((err) => console.log(err))
```

Comment:
Choose either a `console.error` or

```js
.catch(error => {
    throw new Error(error);
})
```

Reply:
Resolved in e0471b3c7d649d657e6d0f4e559786ab6954f785